.github/workflows/run-unit-tests.yml

```diff
@@ -92,6 +92,16 @@ on:
         required: false
         default: 'test'
         description: "Action to pass to xcodebuild when running iOS tests. Defaults to 'test'. Set to '' to just compile and not run tests."
+      enable_all_traits:
+        type: boolean
+        required: false
+        default: false
+        description: "Set to 'false' to enable only default traits when running tests. When true, overrides `disable_default_traits`. Defaults to 'false'."
+      disable_default_traits:
```

Comment:
Ok having thought about this, I've decided this is a bad idea to apply across unit tests. What we should actually have is a job like WASM that runs the tests with `--disable-default-traits` that we can pass some flags into